webpack.config.js

```diff
@@ -1,86 +1,68 @@
-const CopyWebpackPlugin = require('copy-webpack-plugin');
 const path = require('path');
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const webpack = require('webpack');
+
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+const ScratchWebpackConfigBuilder = require('scratch-webpack-configuration');
 
-const base = {
-    mode: process.env.NODE_ENV === 'production' ? 'production' : 'development',
-    devServer: {
-        contentBase: false,
-        host: '0.0.0.0',
-        port: process.env.PORT || 8361
-    },
-    devtool: 'cheap-module-source-map',
-    module: {
-        rules: [
-            {
-                include: [
-                    path.resolve('src')
-                ],
-                test: /\.js$/,
-                loader: 'babel-loader',
-                options: {
-                    presets: [['env', {targets: {browsers: ['last 3 versions', 'Safari >= 8', 'iOS >= 8']}}]]
-                }
+const baseConfig = new ScratchWebpackConfigBuilder({
+    rootPath: path.resolve(__dirname)
+})
+    .enableDevServer(process.env.PORT || 8361)
+    .merge({
+        resolve: {
+            fallback: {
+                Buffer: require.resolve('buffer/')
             }
-        ]
-    },
-    optimization: {
-        minimizer: [
-            new UglifyJsPlugin({
-                include: /\.min\.js$/
-            })
-        ]
-    },
-    plugins: []
-};
+        }
+    })
+    .addPlugin(new webpack.ProvidePlugin({
+        Buffer: ['buffer', 'Buffer']
+    }));
```

Comment:
I'm starting to think this `Buffer` stuff should be in the shared webpack config. What do you think?